coder-sdk/error.go

```diff
@@ -17,31 +17,52 @@ var ErrPermissions = xerrors.New("insufficient permissions")
 // ErrAuthentication describes the error case in which the requester has invalid authentication.
 var ErrAuthentication = xerrors.New("invalid authentication")
 
-// apiError is the expected payload format for our errors.
-type apiError struct {
-	Err apiErrorMsg `json:"error"`
+// APIError is the expected payload format for our errors.
+type APIError struct {
```

Comment:
```suggestion
// APIError is the expected payload format for API errors.
type APIError struct {
```

Reply:
only being particular because making this public reveals its doc comment on godoc